.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Out of curiosity - was .prettierrc a part of some project generating tool or did you prepare it? I'm impressed by just the thought to make it.

Reply:
There's a basic template generated by @nest/cli, I've only added some of my prefered config.